pkg/sfu/buffer/buffer.go

```diff
@@ -445,8 +478,9 @@ func (b *Buffer) doNACKs() {
 		return
 	}
 
-	if r := b.buildNACKPacket(); r != nil {
-		b.feedbackCB(r)
+	if r, numSeqNumsNacked := b.buildNACKPacket(); r != nil {
+		go b.feedbackCB(r)
```

Comment:
Making this a `go` routine. We were making all these feed back calls with buffer lock held. Buffer holds the lock at the top level. But, the TWCC feedback is still done under lock (need to figure out fine-grained locks) as it happens per packet and did not want to spin up a go routine per packet.

Reply:
Agree, during investigate frozen issue, I found these callbacks might slow down buffer.Write that might cause underlying packetio.Buffer full, we can post these op to a channel and use a single goroutine consume it.